scripts.py

```diff
@@ -2540,12 +2538,14 @@ def update_timestamps(self, migration_file):
         if self.overall_timestamp is None:
             return
 
+        if self.overall_timestamp.finish is not None:
+            finish_timestamp = to_utc(self.overall_timestamp.finish)
```

Comment:
TimestampInfo should be responsible for guaranteeing that .start and .finish are timezone-aware and UTC, if they exist.